clients/java-logger/library/src/main/java/com/abcxyz/lumberjack/auditlogclient/processor/CloudLoggingProcessor.java

```diff
@@ -71,6 +73,14 @@ public AuditLogRequest process(AuditLogRequest auditLogRequest) throws LogProces
                           new TypeReference<Map<String, ?>>() {})))
               .setLogName(getLogNameFromLogType(auditLogRequest.getType()))
               .setLabels(auditLogRequest.getLabelsMap())
+              .setOperation(
+                  Operation.of(
+                      auditLogRequest.getOperation().getId(),
+                      auditLogRequest.getOperation().getProducer()))
+              .setTimestamp(
```

Comment:
What if timestamp and operation are null in the `auditLogRequest`? Will we get an exception?

Reply:
Solid point, but it didn't for the java http client case in integration, the bigquery log just shows null for operation. checking...

Reply:
Null cases are covered by protobuf, it is in the genrated code, let me see if I can find a link.